lightning/src/ln/peer_handler.rs

```diff
@@ -1391,7 +1391,7 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 								log_debug!(self.logger, "Received a HandleError event to be ignored for node {}", log_pubkey!(node_id));
 							},
 							msgs::ErrorAction::SendErrorMessage { ref msg } => {
-								log_trace!(self.logger, "Handling SendErrorMessage HandleError event in peer_handler for node {} with message {}",
+								log_gossip!(self.logger, "Handling SendErrorMessage HandleError event in peer_handler for node {} with message {}",
```

Comment:
I think we only want to convert the error handling to things that come from normal gossip operation, which is basically just "we don't support zlib" or "we have already received that message", not things where we send an "error" message, which is usually "something is very bad, and we're closing a channel with you"

Reply:
Let me know if I'm mistaken, but as far as I see right now there is no good way to discern the different `ErrorMessage` types? 
I therefore reverted this back to `log_trace` for the time being with https://github.com/rust-bitcoin/rust-lightning/pull/1145/commits/fbbaeea3afd1f02da29c4b5e7981807b806973ff.

Reply:
Generally, indeed, long-term we want to convert the `IgnoreError` types to `IgnoreAndLog` (mostly), which lets the originator of the error give more instruction here.